src/plugins/interactions/src/systems/interactions/collision_start_event_to_interaction_event.rs

```diff
@@ -0,0 +1,64 @@
+use bevy::prelude::{EventReader, EventWriter};
+use bevy_rapier3d::prelude::CollisionEvent;
+
+use crate::events::InteractionEvent;
+
+pub(crate) fn collision_start_event_to_interaction_event(
+	mut collisions: EventReader<CollisionEvent>,
+	mut interactions: EventWriter<InteractionEvent>,
+) {
+	for collision in collisions.read() {
+		let CollisionEvent::Started(a, b, ..) = collision else {
+			continue;
+		};
+		interactions.send(InteractionEvent::of(*a).with(*b));
+	}
+}
```

Comment:
Could we use this to assign the `ColliderRoot` entity early so that systems consuming the `InteractionEvent` do not need to check the for the root themselves?

Reply:
done